content/source/docs/enterprise/getting-started/workspaces.html.md

```diff
@@ -39,7 +39,9 @@ To create your first workspace, click the "+ New Workspace" button in the upper
 
 On the "Create a new Workspace" page, you need to enter at least two items: a workspace name, and a VCS repository. (You can search for a repo by typing part of its name.) When you've finished, click the "Create Workspace" button.
 
-![fields on new workspace page](./images/work-new-fields.png)
+![fields on new workspace page](../workspaces/images/creating-fields.png)
+
+-> **Note:** You can also create workspaces without a VCS repository, but doing so requires a more advanced workflow for Terraform runs. For more information, see [About Terraform Runs in Terraform Enterprise](../run/index.html).
```

Comment:
I'd say "a different workflow"; at the moment it might be a bit less straightforward, but that's not guaranteed to remain true.